src/styles/card/card.scss

```diff
@@ -0,0 +1,99 @@
+@import "../utils/mixins.scss";
+
+.container {
+  --card-width: 200px;
+  display: grid;
+  row-gap: 46px;
+  column-gap: 48px;
+  grid-template-columns: repeat(4, var(--card-width));
+  max-width: 944px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: --card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 8px;
+  transition: transform 0.3s ease;
+
+  &__image {
+    width: 100%;
+    @include center;
+    margin-bottom: 40px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  /*#region text*/
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments). Fix it everywhere